source/includes/changestream-paradigm-warning.rst

```diff
@@ -0,0 +1,4 @@
+.. warning::
+
+   Attempting to use a ``ChangeStream`` as an ``Iterator`` that is in ``EventEmitter`` mode will cause an error,
```

Comment:
Per style guide, let's use present (simple) tense when possible instead of future ("will cause...").
It could also be helpful to explain why it causes an error.

E.g.
If you access the ``ChangeStream`` in one of the following ways, the driver throws an error:
- As an Iterator in EventEmitter mode
- As an EventEmitter in Iterator mode
< Sentence explaining why the driver throws an error >

Reply:
Thanks. I've simplified the wording around when it will error, and added an explanation